spec/support/stripe_helpers.rb

```diff
@@ -0,0 +1,16 @@
+# -*- encoding : utf-8 -*-
+def signed_headers(signing_secret:, payload:, timestamp: Time.zone.now)
```

Comment:
Ugh required arg with no default is 2.1+

https://makandracards.com/makandra/25739-ruby-2-0-introduces-keyword-arguments